.travis.yml

```diff
@@ -16,18 +16,16 @@ env:
   matrix:
     - PGVERSION=9.5
     - PGVERSION=9.6
+    - PGVERSION=10
 before_install:
   - git clone -b v0.6.2 --depth 1 https://github.com/citusdata/tools.git
   - sudo make -C tools install
   - setup_apt
-  - curl https://install.citusdata.com/community/deb.sh | sudo bash
   - nuke_pg
 install:
   - install_uncrustify
   - install_pg
   - install_custom_pg
-  # download and install HLL manually, as custom builds won't satisfy deps
-  - apt-get download "postgresql-${PGVERSION}-hll=2.10.1.citus-1" && sudo dpkg --force-confold --force-confdef --force-all -i *hll*.deb
```

Comment:
I think I can live with any approach but "leave this as-is", although I prefer not to do that !v10 approach.  Seems too likely that we'll drop the ball and this will bitrot.